src/lib/divider/divider.scss

```diff
@@ -0,0 +1,66 @@
+.mat-divider {
+  display: block;
+  margin: 0;
+  border-top-width: 1px;
+  border-top-style: solid;
+
+  &.mat-divider-vertical {
+    border-top: 0;
+    border-right-width: 1px;
+    border-right-style: solid;
+    position: static;
+    width: auto;
+  }
+
+  &.mat-divider-inset {
+    margin-left: 80px;
```

Comment:
Where does this 80px size come from?  Can you put in a comment explaining it?

Reply:
It came from AngularJS Material found [here](https://github.com/angular/material/blob/72d068593b7f353d32f8308870d2b0d1c7c759c9/src/components/divider/divider.scss#L8). `baseline-grid = 8`.

Reply:
I don't really have an explanation for it, I couldn't find one in the original repo. Is there any way you can reach out to the internal UX team to find verification/justification?